docs/requirements-doc.txt

```diff
@@ -0,0 +1,4 @@
+jupyter-book>=1.0.0
+sphinx>5.0.0
+sphinx_inline_tabs
```

Comment:
`sphinx` is required by `Jupyter-book`, and `sphinx_inline_tabs` is not used, so they can be removed.

```suggestion
```